### PR TITLE
show error message for invalid custom node address

### PR DESCRIPTION
**Pull request by sosan-neikbeen**

### CODE REVIEW

File: src/components/partials/IdentitySearch/CustomNodeModal.vue [hunk 1]
```diff
@@ -102,9 +102,18 @@ export default class CustomNodeModal extends Vue {
...
 
+    validateCustomNodeAddress() {
+        return this.newCustomNodeAddress.startsWith("wss://");
+    }
+
     async saveCustomNode() {
         this.busy = true;
         try {
+            if (!this.validateCustomNodeAddress()) {
+                this.busy = false;
```
Comment (dobschal) [inline src/components/partials/IdentitySearch/CustomNodeModal.vue:113]: @sosan-neikbeen put the ```this.busy = false;``` in a ```finally``` block instead writing it twice.

File: src/components/partials/IdentitySearch/CustomNodeModal.vue [hunk 2]
```diff
@@ -115,6 +124,10 @@ export default class CustomNodeModal extends Vue {
             set<ChainInfo>(StoreKey.CustomNode, customNode);
             this.$emit("save", customNode);
         } catch (e) {
+            if (e.message === "InvalidAddressError") {
+                return (this.error =
+                    "The address is invalid. It should start with 'wss://'.");
+            }
```
Comment (dobschal) [inline src/components/partials/IdentitySearch/CustomNodeModal.vue:130]: @sosan-neikbeen a bit late for an early exit :D Nice, but use a standard if/else statement here.

**Review by dobschal — CHANGES_REQUESTED** [review]

File: src/components/partials/IdentitySearch/CustomNodeModal.vue [hunk 3]
```diff
@@ -115,10 +123,16 @@ export default class CustomNodeModal extends Vue {
...
         } catch (e) {
-            this.error =
-                "Could not reach your entered address. Please check again and enter a valid address.";
+            if (e.message === "InvalidAddressError") {
+                return (this.error =
+                    "The address is invalid. It should start with 'wss://'.");
+            } else {
+                return (this.error =
+                    "Could not reach your entered address. Please check again and enter a valid address.");
+            }
```
Comment (a-oertel) [inline src/components/partials/IdentitySearch/CustomNodeModal.vue:132]: couldn't you write that like:
```
if(e.message === "InvalidAddressError") this.error = "The address is invalid. It should start with 'wss://'.";
else this.error = "Could not reach your entered address. Please check again and enter a valid address.";
```

File: src/components/partials/IdentitySearch/CustomNodeModal.vue [hunk 4]
```diff
@@ -102,9 +102,17 @@ export default class CustomNodeModal extends Vue {
         this.newCustomNodeAddress = this.customNode?.address ?? "";
     }
 
+    validateCustomNodeAddress() {
+        return this.newCustomNodeAddress.startsWith("wss://");
```
Comment (a-oertel) [inline src/components/partials/IdentitySearch/CustomNodeModal.vue:106]: Could you please also add a check for ws:// and allow that? The error message and placeholder can stay as is.

Comment (a-oertel) [inline src/components/partials/IdentitySearch/CustomNodeModal.vue:106]: Could you please also add a check for ws:// and allow that? The error message and placeholder can stay as is.

**Review by a-oertel — APPROVED** [review]

**Pull request merged**